README.md

```diff
@@ -29,6 +29,16 @@ npm run build
 npm run serve
 ```
 
+## Check contents style
+
+Before you commit your changes, you can check the style of the contents by running the following command:
+
+```sh
+$ yarn autocorrect --lint
+```
```

Comment:
The eventmesh-site primarily uses npm. Is it necessary to use yarn here? 

Kindly you can provide a quick user guide link for autocorrect to support contributors who wish to check the format through ways other than the command line.